consult.el

```diff
@@ -3699,6 +3700,74 @@ same major mode as the current buffer are used. See also
   (interactive)
   (consult--imenu (consult--imenu-all-items (consult--imenu-project-buffers))))
 
+;;;;; Command: consult-agenda
+
+(defun consult--org-entries (match scope &rest skip)
+  "Return a list of consult locations from Org entries.
+
+MATCH, SCOPE and SKIP are as in `org-map-entries'."
+  (require 'org)
+  (apply
+   #'org-map-entries
+   (lambda ()
+     (consult--location-candidate
+      (consult--buffer-substring (line-beginning-position)
+                                 (line-end-position)
+                                 'fontify)
+      (point-marker)
+      (current-line)))
```

Comment:
This exact lambda exists in `consult-outline` and should be extracted to a common function `consult--line-candidate`. But maybe you will go with the flattening instead?

Reply:
That's right: this lambda is gone, I'm using Org's headline formatter instead.